hudi-common/src/main/java/org/apache/hudi/common/model/RecordCountAvroPayload.java

```diff
@@ -0,0 +1,199 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.hudi.common.model;
+
+import org.apache.hudi.avro.HoodieAvroUtils;
+import org.apache.hudi.common.util.Option;
+import org.apache.hudi.exception.HoodieException;
+
+import org.apache.avro.AvroRuntimeException;
+import org.apache.avro.Schema;
+import org.apache.avro.generic.GenericRecord;
+import org.apache.avro.generic.IndexedRecord;
+import org.apache.log4j.LogManager;
+import org.apache.log4j.Logger;
+
+import java.io.IOException;
+import java.util.Properties;
+
+/**
+ * Payload clazz that is used for pv/uv.
+ * In order to use 'RecordCountAvroPayload', we need to add field [hoodie_record_count bigint]
+ * to the schema when creating the hudi table to record the result of pv/uv, field 'hoodie_record_count'
+ * does not need to be filled, and flink will automatically set it to "null", "null" represents 1.
```

Comment:
"null" represents 1 is a little strange, can be set to 1 explicitly?

Reply:
I've updated the comment, in fact the `null` is updated to `1` in `#mergeOldRecord` and `#getInsertValue`.